torch_xla/experimental/custom_kernel.py

```diff
@@ -366,3 +385,29 @@ def flash_attention_non_xla(q: torch.Tensor,
   attn_weight = torch.nn.functional.softmax(attn_weight, dim=-1)
   attn_output = attn_weight @ v
   return attn_output
+
+
+@impl(XLA_LIB, "paged_attention", "XLA")
+def paged_attention_xla(q: torch.Tensor, k_pages: torch.Tensor,
+                        v_pages: torch.Tensor, lengths: torch.Tensor,
+                        page_indices: torch.Tensor,
+                        pages_per_compute_block: int):
+  return flash_attention(q, k, v, causal=causal)
```

Comment:
is this a typo? It still call flash_attnetion

Reply:
Yep a typo haha. Updated.